rholang/src/main/scala/coop/rchain/rholang/interpreter/PrettyPrinter.scala

```diff
@@ -91,7 +91,7 @@ case class PrettyPrinter(
       case ESetBody(ParSet(pars, _, _, remainder)) =>
         pure("Set(") |+| buildSeq(pars.sortedPars) |+| buildRemainderString(remainder) |+| pure(")")
       case EMapBody(ParMap(ps, _, _, remainder)) =>
-        pure("{") |+| (pure("") /: ps.sortedMap.zipWithIndex) {
+        pure("{") |+| (pure("") /: ps.sortedList.zipWithIndex) {
```

Comment:
Please make the sortedMap private to prevent similar errors :)

Reply:
Good idea. Done.

Reply:
Or actually not done. More changes required elsewhere.

Reply:
Now it should be done.